src/components/correct-name/CorrectNameRequest.vue

```diff
@@ -96,9 +96,10 @@ import { GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module
 @Component({
   components: {
     ConfirmDialogShared
-  }
+  },
+  mixins: [NameRequestMixin]
```

Comment:
Yup, back to the old way for importing mixins.

Unfortunately, with this way we lose typing of the mixin contents. But that's a problem for another day.

Reply:
Ref: https://github.com/facing-dev/vue-facing-decorator/blob/master/src/mixins.ts

Did you try `toNative(NameRequestMixin)` ? I ask because I think we mostly use class components (ie, our components and mixins look like classes, not the alteration options script).

Reply:
I was not able to use this successfully yet. How about I dive deeper in [18526](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/gh/bcgov/entity/18526) which I will work on next?

Reply:
Yes, perfect. Resolving.

Reply:
See if this helps...?

https://blog.logrocket.com/how-to-use-vue-3-typescript/#using-typescript-mixins-vue